packages/runner/injection/patches/fetch.ts

```diff
@@ -1,16 +1,16 @@
-import { captureFullRequestUrl } from './utils'
+import { captureFullRequestUrl, requestSentWithCredentials } from './utils'
 
-export const patchFetch = (Cypress: Cypress.Cypress, window) => {
+export const patchFetch = (window) => {
   // if fetch is available in the browser, or is polyfilled by whatwg fetch
   // intercept method calls and add cypress headers to determine cookie applications in the proxy
   // for simulated top. @see https://github.github.io/fetch/ for default options
-  if (!Cypress.config('experimentalSessionAndOrigin') || !window.fetch) {
+  if (!window.fetch) {
```

Comment:
experimental session and origin config doesn't really apply here because it is only injected if the flag is enabled anyway